libnavigation-base/src/main/java/com/mapbox/navigation/base/route/NavigationRoute.kt

```diff
@@ -151,19 +155,29 @@ class NavigationRoute internal constructor(
             routerOrigin: RouterOrigin,
             routeParser: SDKRouteParser = NativeRouteParserWrapper
         ): List<NavigationRoute> {
+            logI("NavigationRoute.create called for $routeRequestUrl", TAG)
             return coroutineScope {
                 val deferredResponseParsing = async(ThreadController.DefaultDispatcher) {
-                    DirectionsResponse.fromJson(directionsResponseJson)
+                    logI("parsing directions response to java model from $routeRequestUrl", TAG)
+                    val result = DirectionsResponse.fromJson(directionsResponseJson)
+                    logI("parsed directions response to java model from $routeRequestUrl", TAG)
+                    result
```

Comment:
the same https://github.com/mapbox/mapbox-navigation-android/pull/5965/files#r908327779, let's also add `routeId` there. It can be got from other logs if needed